### ISSUE TITLE
Missing react-a11y converter: react-a11y-role-has-required-aria-props

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#898]
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 2.0.1
-   `typescript-eslint` version: 4.9.1

#### TSLint Rule Name

`react-a11y-role-has-required-aria-props`

Per https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md, the equivalent rule is now `jsx-a11y/role-has-required-aria-props`.

#### Package Source

https://github.com/teamairship/tslint-react-a11y

**Comment by JoshuaKGoldberg** [comment]
Fixed by #1140.

**Comment by rafaelss95** [comment]
I guess this can be closed already.
